devito/ops/types.py

```diff
@@ -64,7 +64,7 @@ class OpsAccess(basic.Basic, sympy.Basic):
     ----------
     base : OpsAccessible
         Symbol to access
-    indices: [int]
+    indices: list of tuples of int
```

Comment:
list of int 
?

Reply:
it's not something like `[1,2,3,4]`
it's more like `[[1,2],[3,4]]` instead

Reply:
list of 2-tuples of int 
?

Reply:
it will depend on the number of dimensions.
it could be `[[1,2,3],[4,5,6]]` if 3D
right, @vicenzo ?

Reply:
It's actuallt a list of `sympy.Integers`

Reply:
the PR is 99% OK apart from this minor thing. Since there are too many open PRs, Im gonna merge it as is now. Please fix this in the next OPS PR

Reply:
ok